package.json

```diff
@@ -32,7 +32,8 @@
     "connect": "^3.2.0",
     "js-yaml": "^3.3.0",
     "swagger-tools": "0.10.1",
-    "unzip": "^0.1.11"
+    "unzip": "^0.1.11",
+    "natural": "0.5.0"
```

Comment:
Not sure yet, if we're using natural (porter algorithm) or the porter2, which group2 uses.